.github/workflows/narwhal.yml

```diff
@@ -1,15 +1,6 @@
 name: Narwhal
```

Comment:
why not remove the whole workflow?

Reply:
Definitely we will remove it later. But I want to leave the workflow to run nightly for a few more days.